tsup.config.ts

```
@@ -28,6 +28,10 @@ export default defineConfig({
     'Text/index': 'src/components/Text/Text.tsx',
     'TextArea/index': 'src/components/TextArea/TextArea.tsx',
     'Toast/index': 'src/components/Toast/Toast.tsx',
+    'ProgressBar/index': 'src/components/ProgressBar/ProgressBar.tsx',
```

ESLint

[error] Duplicate key 'ProgressBar/index'.

(no-dupe-keys)